### PR TITLE
added retrospectives.md to coursebook/general

### LINKED ISSUE #158: Add document to coursebook/general on how to run retrospectives
A document detailing how we run classroom stop-go-continue's and team retrospectives should be added to the coursebook/general folder. Including location of where minutes should be stored.

### PR DESCRIPTION
**Pull request by bradreeder**
* Created retrospectives.md file detailing how we run stop-go-continues and team retrospectives.

Fixes #158

### CODE REVIEW

File: coursebook/general/retrospectives.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+The feedback can range broadly from environmental concerns, to timetabling, to feedback on workshops.
+
+Once students have filled a whiteboard with their feedback, an assigned facilitator will read each point and they are discussed by the group. Every student votes at the end on a single "stop", "go", and "continue" item they consider the highest priority, and those with the most consensus are recorded as key action points.
+
+The minutes of these meetings are captured ([London](https://github.com/foundersandcoders/london-curriculum), [Nazareth](https://github.com/foundersandcoders/nazareth-curriculum)), forwarded to mentors of future weeks, and used when we come to prepare for the next curriculum so that we can improve it as a group.
+
+## Project retrospectives
+
+We also run a smaller "stop go continue" meeting with each [student team](./teams.md) so they are able to reflect on how their group project work went that week.
```
Comment (jsms90) [inline coursebook/general/retrospectives.md]: You call the whole document "retrospectives". Then you call the cohort version "stop-go-continue". Then you call the team version "project retrospectives", but within this title it is referred to as "a smaller s-g-c".

The naming makes it really confusing to understand which thing you are talking about when you say "s-g-c" and which thing you are talking about when you say "retrospective".

**Comment by bradreeder** [comment]
Note to self: update [retrospectives](https://github.com/FAC10/retrospectives) repo when done.

**Comment by jsms90** [comment]
Sorry for the merge. Momentarily forgot which repo I was in, since I was a "reviewer". Oops :-1:

**Comment by bradreeder** [comment]
I take it by Nelson's thumb and Eoin's approval it had two reviews so hope the others don't mind. 👍

**Comment by des-des** [comment]
@jsms90 dw thnx for review!

**Pull request merged**